auth-web/src/models/business.ts

```diff
@@ -94,16 +94,18 @@ export interface BusinessRequest {
             name: string,
             accountId: number
         },
-        business: {
+        business?: {
```

Comment:
When is this optional?

Reply:
Made it optional to support IA filing (which still use business object) for now.

Reply:
Please add a comment saying just that.